saleor/graphql/invoice/types.py

```diff
@@ -0,0 +1,23 @@
+import graphene
+
+from ...invoice import models
+from ..core.connection import CountableDjangoObjectType
+from ..core.types.common import InvoiceJobInterface
+
+
+class Invoice(CountableDjangoObjectType):
+    class Meta:
+        description = "Represents an Invoice."
+        interfaces = [graphene.relay.Node]
```

Comment:
Model inherit form `ModelWithMetadata` and support metadata. 
We should add the proper metadata interface to API or remove the metadata form model.